lib/modules/apostrophe-areas/public/js/editor.js

```diff
@@ -673,13 +689,18 @@ apos.define('apostrophe-areas-editor', {
           options: options,
           type: type
         }, function(html) {
+          apos.ui.globalBusy(false);
           // This rather intense code works around
           // various situations in which jquery is
           // picky about HTML
           var $newWidget = $($.parseHTML($.trim(html), null, true));
           self.replaceWidget($widget, $newWidget);
+          return callback && callback(null);
           // TODO should this take a callback?
           // return callback(null);
+        }, function(err) {
+          apos.ui.globalBusy(false);
+          return callback && callback(err);
```

Comment:
If there's no callback this will error very quietly. Maybe we should log here as well.

Reply:
I will make a note to add this as a follow-on.